torch2jax/__init__.py

```diff
@@ -618,7 +641,10 @@ def scatter_add(input, dim, index, src):
 
 @implements(torch.sort, out_kwarg=True, Torchish_member=True)
 def sort(input, dim=-1, descending=False, stable=False):
-  return jnp.sort(_v(input), axis=dim, stable=stable, descending=descending)
+  input = _v(input)
+  sorted_indices = jnp.argsort(input, axis=dim, stable=stable, descending=descending)
+  sorted_values = jnp.take_along_axis(input, sorted_indices, axis=dim)
+  return torch.return_types.sort([sorted_values, sorted_indices])
```

Comment:
this changes the return type, so the previous impl was wrong? do we not have tests for sort?